README.md

```diff
@@ -1,5 +1,29 @@
 [![CircleCI](https://circleci.com/gh/AndelaOSP/art-dashboard.svg?style=svg)](https://circleci.com/gh/AndelaOSP/art-dashboard)
-# art-dashboard
+# ART
 [![Coverage Status](https://coveralls.io/repos/github/AndelaOSP/art-dashboard/badge.svg)](https://coveralls.io/github/AndelaOSP/art-dashboard)
 
 This is the admin dashboard for the ART application
+
+## Directory Structure
+```bash
+├── src
+│   ├── _test_
+│   │   ├── **/*.css
+│   ├── _actions
+│   ├── _components
+│   ├── _constants
+│   ├── _reducers
+│   └── components
+```
+ ### Components Folder (components)
+> Contains presentational components that are also within their specific folders. e.g. AssetsComponent is in `components/Assets`
+### Containers Folder (_components)
+ > Contains container components that are also within their specific folders. e.g. AddAssetContainer is in `_components/Assets`
```

Comment:
Instead of `that are also within`, just have `within`. **_"that are also_"** are filler words in this case.


Also, capitalize "e" in` e.g.`. It follows a full stop, therefore, the first letter should be a capital